src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -139,9 +125,12 @@ public static void main(final String[] args) {
       usage();
       return;
     }
-    SwingUtilities.invokeLater(LookAndFeel::setupLookAndFeel);
 
-    HttpProxy.setupProxies();
+    LookAndFeel.setupLookAndFeel();
```

Comment:
Interesting one here. JFX is still a beta feature, it should also play well with this call as well (JFX was built to be able to integrate into existing Swing components). I think it is the better approach to leave the common path as unaltered as we can for the moment. Considering the case of many beta features, if we optimistically locked out all code that might not run, we would likely be over-complicating things. 

In this case specifically, we would remove `LookAndFeel.java` probably completely when JavaFx comes along, beyond sweeping for Swing code to remove, I'm pretty confident we will not forget to remove the call here.